cmd/ingester/main.go

```diff
@@ -63,10 +65,12 @@ func main() {
 		&schemaConfig, &ingesterConfig, &clientConfig, &limits, &preallocConfig)
 	flag.UintVar(&maxStreams, "ingester.max-concurrent-streams", 1000, "Limit on the number of concurrent streams for gRPC calls (0 = unlimited)")
 	flag.IntVar(&eventSampleRate, "event.sample-rate", 0, "How often to sample observability events (0 = never).")
+	flag.BoolVar(&marshalFullSize, "ingester.fullsize-chunks", true, "When saving varbit chunks, pad to 1024 bytes")
 	flag.Parse()
 
 	util.InitLogger(&serverConfig)
 	util.InitEvents(eventSampleRate)
+	encoding.AlwaysMarshalFullsizeChunks = marshalFullSize
```

Comment:
Could this be done more like PreallocConfig in the client pkg? https://github.com/cortexproject/cortex/blob/master/pkg/ingester/client/timeseries.go#L13